charts/rasa-x/templates/rasa-x-deployment.yaml

```diff
@@ -90,6 +90,13 @@ spec:
             secretKeyRef:
               name: {{ template "rasa-x.secret" . }}
               key: "rasaXToken"
+        - name: "RASA_X_USERNAME"
+          value: "{{ $.Values.rasax.initialUser.username }}"
```

Comment:
should work like this, no?
```suggestion
          value: "{{ .Values.rasax.initialUser.username }}"
```

Reply:
Hm, it worked with $ as well. But will change it